UITests/CheckmarxTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using FlaUI.Core.AutomationElements;
+using System.Threading.Tasks;
+
+namespace UITests
+{
+    [TestClass]
+    public class CheckmarxTests : BaseTest
+    {
+        [ClassInitialize]
+        public static void ClassInit(TestContext context)
+        {
```

Comment:
this method just calls ClassInitialize from the base class, which is a bit redundant. You can directly use the base class's ClassInitialize method without redefining it